src/WooCommerce.php

```diff
@@ -539,8 +539,19 @@ public static function woocommerce_get_item_data($data, $cartItem) {
    */
   public static function woocommerce_display_item_meta($html, $item, $args) {
     $strings = [];
+    $data = [];
```

Comment:
The separator could be added first to reduce the amount things being juggled here (and juggle them in their actual order).

```suggestion
    $data = [];
    // Add a separator before product attributes.
    $data[] = [
      'name' => '',
      'value' => '<hr>',
    ];
```

Reply:
It also seems like you could easily get rid of the multidimensional array_merge:

```suggestion
    $data = static::getProductData($product);
    // Add a separator before product data and attributes.
    $data[] = [
      'name' => '',
      'value' => '<hr>',
    ];
```

The same probably applies to the delivery time splicing in between, although I still did not really understand yet into which exact position it is sliced into.